packages/veritone-react-common/src/components/TranscriptEngineOutput/TranscriptSegment/SnippetSegment/index.js

```diff
@@ -72,20 +73,20 @@ export default class SnippetSegment extends Component {
       content,
       editMode,
       contentClassName,
-      selectedEngineId,
       startMediaPlayHeadMs,
       stopMediaPlayHeadMs
     } = this.props;
 
     const contentComponents = content.fragments.map((entry, index) => {
       const startTime = entry.startTimeMs;
       const stopTime = entry.stopTimeMs;
+      const fragmentKey = entry.guid ?
+        entry.guid :
+        `snippet-fragment-${startTime}-${stopTime}-${entry.value ? entry.value.substr(0, 32) : ''}`;
```

Comment:
would prefer to see `const value = entry.value || ''`

Reply:
done